apps/web/components/apps/AppList.tsx

```diff
@@ -24,16 +25,29 @@ import {
 
 import AppListCard from "@components/AppListCard";
```

Comment:
AppListCard also need to be moved to features package after moving AppList to features package